src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -224,6 +224,7 @@ list of available suffixes.
 | Longitudinal relaxation rate map                       | R1map     | In seconds<sup>-1</sup> (1/s). R1 maps are REQUIRED to use this suffix irrespective of the method they are related to.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | True transverse relaxation rate map                    | R2map     | In seconds<sup>-1</sup> (1/s). R2 maps are REQUIRED to use this suffix irrespective of the method they are related to.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | Observed transverse relaxation rate map                | R2starmap | In seconds<sup>-1</sup> (1/s). R2* maps are REQUIRED to use this suffix irrespective of the method they are related to.                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| Observed signal amplitude map                          | S0map | In arbitrary units (a.u.). For a multi-echo sequence, S0 maps index the exponential of the intercept for a linear decay model across log-transformed echos. For more information, please see <https://tedana.readthedocs.io/en/latest/approach.html#monoexponential-decay-model-fit>. S0 maps are REQUIRED to use this suffix irrespective of the method they are related to. Associated suffixes: T2starmap                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
Sorry for the possibly silly question - is the `<...>` formatting a valid link for markdown? I would have expected to see `[]()` formatting?

The text looks great though. I'm happy with it!

Reply:
Sorry, I was just making sure it was appearing as a hyperlink ! But linking out some plain text is easier on the eyes, so I updated to switch that. Let me know what you think :smile_cat:

Reply:
@emdupre thank you for the addition! I have a minor wording concern, of course because of my writing :) 

For the suffixes that group together input files I used `Associated output suffixes` (Parent--> Child relationship) to denote the maps they can generate. You use `Associated suffixes` (Sibling relationship) here. I was just wondering if these relationships are easy to perceive with the current text.

Reply:
I'm probably the worst person to evaluate how easy it is to read, unfortunately, since I wrote it :sweat_smile: But it'd be great to hear feedback ! Happy to update.